SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -382,4 +373,24 @@ QueryConstraint InterestFactory::CreateLevelConstraints() const
 	return LevelConstraint;
 }
 
+TArray<uint32> InterestFactory::CreateClientResultType() const
```

Comment:
Build this each time for now- once @MatthewSandfordImprobable 's refactor is in this can be cached in the interest factory instance.

Reply:
refactor not going in, maybe should set a static variable the same way the checkout radius constraint is set

Reply:
(seems a little ugly however, so open to input on what to do)

Reply:
We should definitely cache this 👍

Reply:
cached in the same way the checkout radius constraint is